website/hw1.md

```diff
@@ -0,0 +1,4 @@
+@def sequence = [""]
```

Comment:
I wonder if we should start having some kind of directory structure under `website/` since it's becoming a bit unruly right now.

Reply:
i think you just need better names: week1 and abstraction can be called section1.1 and section1.2, and then it will be clear.

Directory structure will also mean longer urls.

# REMEMBER to create redirect pages!

If you rename week1.md to section1.1.md, then you need to create a file called week1.md with contents:

```
This page was moved [here](/section1.1/)
```

Reply:
will do a in a follow up PR if we decide on this.